CHANGELOG.md

```diff
@@ -2,6 +2,8 @@ This project follows [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+* Remove `optional` setting. To update find all `optional: true` and change to `default: nil` instead.
```

Comment:
It may be worth mentioning that it's a breaking change :thinking: Like:
```suggestion
* Breaking change: Remove `optional` setting. To update find all `optional: true` and change to `default: nil` instead.
```